src/components/timeline/Extensionright.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box, Wheel, Line, RightLine } from './Extensionboth';
+import { MobileContainer, Mobile, Desktop, RightContainer, VerticalWrapper } from './styles';
+
+export const VerticalLine = () => (
+  <VerticalWrapper>
+    <div />
+  </VerticalWrapper>
+);
+export default function ExtensionRight({ position, wheelcolor, node, boxcolor }) {
+  return (
+    <>
+      <Desktop>
```

Comment:
Change to `DesktopView` and `MobileView` respectively

Reply:
Done